ntc_templates/templates/arista_eos_show_ip_bgp_detail.textfsm

```diff
@@ -32,6 +33,7 @@ Start
   ^\s+Origin\s+${ORIGIN},\s+metric\s+${METRIC},\s+localpref\s+${LOCAL_PREF},\s+IGP\s+metric\s+\S+,\s+weight\s+${WEIGHT},\s+received\s+\S+\s+ago,\s+${VALID},\s+${PATH_TYPE},\s+atomic-aggregate
   ^\s+Origin\s+${ORIGIN},\s+metric\s+${METRIC},\s+localpref\s+${LOCAL_PREF},\s+IGP\s+metric\s+\S+,\s+weight\s+${WEIGHT},\s+received\s+\S+\s+ago,\s+${VALID},\s+${PATH_TYPE}
   ^\s+Community:\s+${COMMUNITY}$$
+  ^\s+Large Community:\s+${LARGE_COMMUNITY}$$
```

Comment:
For the sake of flexibility and future proofing, I'm suggesting we replace a literal space with a regex.

To accept my suggestion, please click the "commit suggestion" button. Thank you!

```suggestion
  ^\s+Large\s+Community:\s+${LARGE_COMMUNITY}$$
```

Reply:
I missed this one. Thanks!